radontea/_alg_art.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 
 
-def art(sinogram, angles, initial=None, iterations=1,
-        count=None, max_count=None):
+def art(sinogram: np.ndarray, angles: np.ndarray, initial: np.ndarray = None,
+        iterations: int = 1, count=None, max_count=None) -> None:
```

Comment:
Shouldn't `-> None` be `-> np.ndarray`? I should probably add a mypy type check in CI.

Reply:
If you add mypy there would be a problem that mypy doesn't know numpy types without https://github.com/predictive-analytics-lab/data-science-types with is not a complete implementation of the numpy library

Reply:
Ok thanks for that information. Then I won't do it.